selfdrive/test/process_replay/regen.py

```diff
@@ -268,11 +270,18 @@ def regen_and_save(route, sidx, upload=False, use_route_meta=False):
     lr = LogReader(r.log_paths()[args.seg])
     fr = FrameReader(r.camera_paths()[args.seg])
   else:
-    lr = LogReader(f"cd:/{route.replace('|', '/')}/{sidx}/rlog")
+    lr = LogReader(f"cd:/{route.replace('|', '/')}/{sidx}/rlog.bz2")
```

Comment:
we could also regenerate all the routes, but then we upload and store 40MB per segment instead of 10-13, which also takes much longer to upload